### PR TITLE
0.0.4

### PR DESCRIPTION
**Pull request by Arzte**
Its about time we bump the version, TODO stop slacking on version updates.

### CODE REVIEW

**Comment by Dougley** [comment]
Bump the version in package.json too.

**Comment by Arzte** [comment]
👍

File: Commands/Commands/utils.js [hunk 1]
```diff
@@ -87,6 +87,8 @@ commands.stats = {
           },
           fields: field
         }).catch(bugsnag.notify) // Send Message to DM error
+      }).then(successmsg => {
+        setTimeout(() => bot.Messages.deleteMessages([msg]), config.timeouts.messageDelete)
```
Comment (Dougley) [inline Commands/Commands/utils.js]: Can't bulk-delete a single message.
If this is the `!stats` message, just delete it instantly.

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/utils.js (hunk 1 above)
Comment (Dougley) [reply Commands/Commands/utils.js]: If this is the result that got send to your DMs,  deleting isn't necessary.

File: package.json [hunk 2]
```diff
@@ -15,7 +15,14 @@
   },
   "maintainers": [
     {
-      "name": "Benjamin Hoffmeyer (TheDoctorsLife)"
+      "name": "Remco Jongschaap (SteamingMutt)",
```
Comment (Dougley) [inline package.json]: I can't be both the author and a maintainer 😉

Comment (Arzte) [reply package.json]: 👀

**Review by Dougley — APPROVED** [review]

**Pull request merged**